### PR TITLE
Feature/signage dashboard

### PR DESCRIPTION
**Pull request by BertCly**
new menu item for public display dashboard

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -19,5 +20,9 @@
 Route::redirect('/', '/dashboard');
 
 Route::get('/dashboard',[DashboardController::class, 'dashboard'])->middleware(['auth', 'verified'])->name('dashboard');
+Route::get('/display',[DisplayController::class, 'showDisplayPrivate'])->middleware(['auth', 'verified'])->name('displays.private.show');
```
Comment (RosiersRobin) [inline routes/web.php:23]: CS

**Review by RosiersRobin — CHANGES_REQUESTED** [review]
I've added some suggestions.

File: app/Http/Controllers/OrderController.php [hunk 2]
```diff
@@ -2,23 +2,44 @@
...
 use App\Models\Product;
 use Carbon\Carbon;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
 
 class OrderController extends Controller
 {
 
-    public function getOrders()
+    public function getOrders(Request $request)
```
[the remark this answers is not in the document]
Comment (BertCly) [reply app/Http/Controllers/OrderController.php:15]: was not in pull request, but will update..

File: app/Http/Controllers/OrderController.php [hunk 3]
```diff
@@ -2,23 +2,44 @@
...
     {
-        $orders = Order::getOrders($this->getDate())->get();
+        if(Auth::check()){
+            $company = Auth::user()->company;
+        }else{
+            $company = Company::query()->where('token', $request->input('company_token'))->firstOrFail();
+        }
+        $orders = Order::getOrders($company, $this->getDate())->get();
```
[the remark this answers is not in the document]
Comment (BertCly) [reply app/Http/Controllers/OrderController.php:22]: not part of the PR. please make new PR

File: app/Http/Controllers/OrderController.php [hunk 4]
```diff
@@ -2,23 +2,44 @@
...
     }
 
+    public function getSelectedRunner(Request $request)
+    {
+        if(Auth::check()){
+            $company = Auth::user()->company;
+        }else{
+            $company = Company::query()->where('token', $request->input('company_token'))->firstOrFail();
+        }
+        $order = Order::getOrders($company, $this->getDate())->first();
```
[the remark this answers is not in the document]
Comment (BertCly) [reply app/Http/Controllers/OrderController.php:36]: not part of the PR. please make new PR

File: app/Models/Order.php [hunk 5]
```diff
@@ -25,12 +25,15 @@ public function product()
         return $this->belongsTo(Product::class);
     }
 
-    public static function getOrders($date)
+    public static function getOrders(Company $company, $date)
```
[the remark this answers is not in the document]
Comment (BertCly) [reply app/Models/Order.php:28]: make new PR for this

**Review by RosiersRobin — APPROVED** [review]

**Pull request merged**